dvc/command/ls/ls_colors.py

```diff
@@ -0,0 +1,54 @@
+import os
+
+
+class LsColors(object):
+    default = "rs=0:di=01;34:ex=01;32"
+
+    def __init__(self, lscolors=None):
+        self._extensions = {}
+        self._codes = {}
+        if lscolors:
+            self.load(lscolors)
+        else:
+            self.load(os.environ.get("LS_COLORS", LsColors.default))
+
+    def load(self, lscolors):
+        if not lscolors:
+            return
```

Comment:
> Do you think that converting load -> _load would help us get rid of double-check?

I think it make sense, I wanted to provide API for loading custom dict, but after iterative changes it isn't used anywhere

> instead of return, go for self.load

Agreed, it looks much more nicer! Thank you!